src/scene/gsplat/gsplat-data.js

```diff
@@ -563,7 +563,7 @@ class GSplatData {
             data.f_dc_0[i] = (c.x - 0.5) / SH_C0;
             data.f_dc_1[i] = (c.y - 0.5) / SH_C0;
             data.f_dc_2[i] = (c.z - 0.5) / SH_C0;
-            data.opacity[i] = -Math.log(1 / c.w - 1);
+            data.opacity[i] = (c.w <= 0) ? -40 : (c.w >= 1) ? 40 : -Math.log(1 / c.w - 1);
```

Comment:
it'd be good to comment on what -40 and 40 represents

Reply:
Sure, good point thanks.

Reply:
TBH I think it was a mistake to store scale & opacity values as exponents/sigmoid values the way they're stored in the source PLY data. We should be converting these to linear/direct values at load time, once, and storing these in the compressed format.